src/cargo_toml.rs

```diff
@@ -144,34 +104,22 @@ struct CargoTomlMetadata {
 /// Those fields are the part of the
 /// [python core metadata](https://packaging.python.org/specifications/core-metadata/)
 /// that doesn't have an equivalent in cargo's `[package]` table
-#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Default)]
```

Comment:
I don't think we need `Eq`/`PartialEq` in maturin, and `HashMap<String, toml_edit::easy::Value>` prevents deriving them.